azure-iot-device/tests/common/transport/test_pipeline_events_base.py

```diff
@@ -0,0 +1,15 @@
+# -------------------------------------------------------------------------
+# Copyright (c) Microsoft Corporation. All rights reserved.
+# Licensed under the MIT License. See License.txt in the project root for
+# license information.
+# --------------------------------------------------------------------------
+import pytest
+from azure.iot.device.common.transport import pipeline_events_base
+
+
+@pytest.mark.describe("PipelineEvent object")
+class TestPipelineEvent(object):
+    @pytest.mark.it("Sets arguments correctly")
```

Comment:
>"Sets arguments correctly" [](start = 20, length = 26)

As in the other comment, this is too vague to be a requirement IMO (and everywhere it occurs)